src/message/poll.rs

```diff
@@ -38,13 +39,16 @@ pub enum MessageData {
     /// Data under the &lt;host:infData&gt; tag
     #[serde(rename = "infData")]
     HostInfo(HostInfoResponseData),
+    /// Data under the &lt;lowbalance&gt; tag
+    #[serde(rename = "pollData")]
+    LowBalance(LowBalance),
```

Comment:
The rgp-poll extension also uses `pollData` (https://www.verisign.com/assets/epp-sdk/verisign_epp-extension_rgp-poll_v01.html). It's unfortunate that `quick-xml` drops the namespaces here.

I wonder if we could use the xmlns attribute to support multiple different extensions, if that isn't dropped. So instead of the `LowBalance` variant here, we would have a `PollData` variant, which again is an enum which is untagged and uses the xmlns attribute as a discriminator.

Reply:
Do we have an urgent need for the rgp poll stuff? All of this is just more reason to finish my instant-xml port...

Reply:
You will probably get `rgp-poll:pollData` responses once you submit a restore command. Thus, this might not be very urgent. But I thought maybe it makes sense to keep that in mind, to keep the API a little bit more stable. But I guess it makes sense to look at this again once we/you replace `quick-xml`.